sql/moz-fx-data-shared-prod/static/country_codes_v1/schema.json

```diff
@@ -28,5 +28,17 @@
     "description": "The UNSD intermediate region name.",
     "type": "STRING",
     "mode": "REQUIRED"
+  },
+  {
+    "name": "pocket_available_on_newtab",
+    "description": "Whether Pocket is available on the newtab page in this country.",
+    "type": "BOOL",
+    "mode": "REQUIRED"
+  },
+    {
+    "name": "mozilla_vpn_available",
+    "description": "Whether Mozilla VPN is available in this country.",
+    "type": "BOOL",
+    "mode": "REQUIRED"
```

Comment:
Some thoughts:
- It feels incongruous to add these fields to a "country codes" table, but renaming it "countries" would be a bit of a hassle (we'd probably have to create a `static.country_codes_v1` view that selects from the new table to avoid breaking stuff, at least for a period of time).
- With this setting a precedent I'd be leery about this table getting scope creep over time as people find additional miscellaneous country-specific data they want to have available.
- I wonder if putting this data in static tables in the `pocket_derived` and `mozilla_vpn_derived` datasets would be better.  That would help with discoverability, and make it more reasonable to include other related data that might also be useful (e.g. the date it became available in that country, or the release "wave" numbers for VPN).

Reply:
> It feels incongruous to add these fields to a "country codes" table, but renaming it "countries" would be a bit of a hassle ...

Completely agree here, I think we will want to just go ahead and rename this table to `countries` and fix the dependencies. Worth it for comprehension's sake. It's a small table, the biggest worry would be backfills of larger tables I think. 

RE: 2 and 3 - adding country specific data to one table is intentional. That way we can break usage/revenue/etc. down by increasingly more dimensions. In this format we are limited to current values, agreed that anything that maintains history would need a different table grain and should probably be in those datasets